backend/de.metas.contracts/src/main/java/de/metas/contracts/commission/commissioninstance/businesslogic/sales/commissiontrigger/mediatedorder/MediatedOrderFactory.java

```diff
@@ -74,9 +81,9 @@ public MediatedOrderFactory(@NonNull final CommissionProductService commissionPr
 	 {
 		 final DocStatus orderDocStatus = DocStatus.ofCode(mediatedOrder.getDocStatus());
 
-		 if (!orderDocStatus.isCompleted())
+		 if (!hasTheRightStatus(orderDocStatus))
```

Comment:
i think we can replace `hasTheRightStatus` method with orderDocStatus.isInProgressCompletedOrClosed()

Reply:
yeah, we could do that... I saw the method as an opportunity to make a statement.... but yeah, I can change it

Reply:
i think I found a good reason for the dedicated method..